biocrnpyler/chemical_reaction_network.py

```diff
@@ -280,8 +280,8 @@ def generate_sbml_model(self, stochastic_model=False, show_warnings = False, **k
         :param keywords: extra keywords pass onto create_sbml_model() and add_all_reactions()
         :return: tuple: (document,model) SBML objects
         """
-        ChemicalReactionNetwork.check_crn_validity(self._reactions, self._species, show_warnings=show_warnings)
-
+        if(check_validity):
+            ChemicalReactionNetwork.check_crn_validity(self._reactions, self._species, show_warnings=show_warnings)
```

Comment:
can you add one empty line after this for readablity? Thanks!

Reply:
![Screenshot 2021-06-06 at 19 37 44](https://user-images.githubusercontent.com/17138838/120936225-ac524e00-c6fe-11eb-8858-d1769f022e34.png)

Reply:
i added `check_validity` for `simulate_with_roadrunner()`